script.bluetooth.delay/Device2.py

```diff
@@ -0,0 +1,168 @@
+import xbmc
+from xml.dom import minidom
+import xbmcaddon
+import xbmcgui
+import os
+import time
+import xbmcvfs
+
+import sys
+if sys.version > '3':
+    xbmc.translatePath = xbmcvfs.translatePath
+
+integer_types = (int)
+
+Addon = xbmcaddon.Addon('script.bluetooth.delay')
+
+line1 = Addon.getSetting('line1')
+line2 = Addon.getSetting('line2')
+t = 1000
+
+if (xbmc.getCondVisibility('Player.HasMedia') == False):
+    xbmc.executebuiltin('Notification(%s, %s, %d)'%("",Addon.getLocalizedString(30015), t))

```

Comment:
this should be xbmcgui.Dialog().notification()
We have support for this on the python api no need for builtins